MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/kits/basekits/Kit.java

```diff
@@ -122,8 +153,37 @@ public void setDescription(List<String> description) {
 
   public abstract ItemStack getItemStack();
 
-  public abstract void giveKitItems(Player player);
+  public void giveKitItems(Player player) {
+    player.getInventory().clear();
+    kitItems.forEach((item, slots) -> {
+      for (Integer slot : slots) {
+        switch (slot) {
+          case (-5) -> {
+            player.getInventory().setHelmet(item);
+          }
+          case (-4) -> {
+            player.getInventory().setChestplate(item);
+          }
+          case (-3) -> {
+            player.getInventory().setLeggings(item);
+          }
+          case (-2) -> {
+            player.getInventory().setBoots(item);
+          }
+          case (-1) -> {
+            player.getInventory().setItem(XItemStack.firstEmpty(player.getInventory(), 0), item);
+          }
+          default -> {
```

Comment:
Same here, using enhanced switch expression will broke legacy versions.

Reply:
Resolved, thanks for pointing out.